tests/python/relax/test_transform_codegen_pass.py

```diff
@@ -40,7 +40,7 @@
 )
 
 # Global variable in pytest that applies markers to all tests.
-pytestmark = [has_tensorrt_codegen, has_tensorrt_runtime]
+# pytestmark = [has_tensorrt_codegen, has_tensorrt_runtime]
```

Comment:
I guess we will need this

Reply:
Whoops, thank you for the catch.  This was commented out for debugging, and the PR is updated to remove the change.